kie-server-parent/kie-server-controller/kie-server-controller-test-war/src/main/assembly/assembly-ee6-container.xml

```diff
@@ -50,6 +50,7 @@
 
         <include>com.sun.xml.bind:jaxb-impl</include>
         <include>com.sun.xml.bind:jaxb-core</include>
+        <include>org.slf4j:slf4j-jdk14</include>
```

Comment:
It should be generally enough to include just the slf4j-api. The logging impl should be then supplied by the container. 

Although I am wondering if there isn't something weird going on here, because slf4j-jdk14 does _not_ contain the org.slf4j.Logger class, that one is indeed in slf4j-api.